server/graphql/v1/mutations/orders.js

```diff
@@ -225,15 +226,16 @@ const checkAndUpdateProfileInfo = async (order, fromAccount, isGuest, currency)
 async function checkCaptcha(order, remoteUser, reqIp) {
   const requestedProvider = order.guestInfo?.captcha?.provider;
   const isCaptchaEnabled = parseToBoolean(config.captcha?.enabled);
+  const isCreditCardOrder = order.paymentMethod.type === PAYMENT_METHOD_TYPE.CREDITCARD;
```

Comment:
Should we be defensive on `order.paymentMethod` being null?

I don't know here but in some cases it's null for Bank Transfers.

Reply:
True. I'll add a conditional operator here.